### PR TITLE
Implement Hometask 3 OOP

### PR DESCRIPTION
**Pull request by StanchykAF**

### CODE REVIEW

File: src/main/java/com/epam/javacourse/hometask3/Library.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        for (Book book : books) {
+            if(book.getPublisher().equals(publisher)) {
+                System.out.println(book);
+            }
+        }
+    }
+
+    public static void filter(Book[] books, boolean paperback) {
+        for (Book book : books) {
+            if(book.isPaperback() && paperback) {
```
Comment (AndreReen) [inline src/main/java/com/epam/javacourse/hometask3/Library.java]: We will never be able to fetch a book when paperback param is false

File: src/main/java/com/epam/javacourse/hometask3/Library.java [hunk 2]
```diff
@@ -84,7 +84,7 @@ public static void filter(Book[] books, Publisher publisher) {
 
     public static void filter(Book[] books, boolean paperback) {
         for (Book book : books) {
-            if(book.isPaperback() && paperback) {
+            if((book.isPaperback() && paperback) || (!book.isPaperback() && !paperback)) {
```
Comment (AndreReen) [inline src/main/java/com/epam/javacourse/hometask3/Library.java]: We can simplify this conditional statement to if(book.isPaperback() == paperback)

Comment (StanchykAF) [reply src/main/java/com/epam/javacourse/hometask3/Library.java]: Yeah, you're right here. I missed it

**Review by AndreReen — APPROVED** [review]
:thumbsup:

**Pull request merged**